.github/workflows/publish-packages.yml

```
@@ -69,38 +64,82 @@ jobs:
+      - name: Install additional build dependencies
         if: startsWith(matrix.build.OS, 'ubuntu')
+        shell: bash
         run: |
           sudo apt-get update
-          sudo apt-get install -y libclang-dev protobuf-compiler cmake pkg-config libssl-dev ca-certificates openssh-client libudev-dev
+          sudo apt-get install -y cmake pkg-config libssl-dev
+
+      - name: Install macOS dependencies
+        if: startsWith(matrix.build.OS, 'macos')
+        shell: bash
+        run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2086:info:4:40: Double quote to prevent globbing and word splitting

(shellcheck)